Sprint-1/explore/chrome.md

```diff
@@ -11,8 +11,12 @@ In the Chrome console,
 invoke the function `alert` with an input string of `"Hello world!"`;
 
 What effect does calling the `alert` function have?
+A pop-up alert appear with "Hello world!"
 
 Now try invoking the function `prompt` with a string input of `"What is your name?"` - store the return value of your call to `prompt` in an variable called `myName`.
 
 What effect does calling the `prompt` function have?
+a prompt dialog appear in the browser with "What is your name?", input field where the user can type a respond, it has 2 buttons: Ok and Cancel, the text is stored in the variable `myName`
+
 What is the return value of `prompt`?
+the text the user enters returned as a string
```

Comment:
If you were writing a program that uses `prompt()` to ask for an input value, how do you check if the user clicked "OK" or "Cancel"?